internal/agent/upgrade.go

```diff
@@ -41,7 +41,10 @@ func ListReleases() []string {
 	return releases
 }
 
-func Upgrade(version, image string, force, debug, strictValidations bool, dirs []string) error {
+func Upgrade(
+	version, image string, force, debug, strictValidations bool, dirs []string,
```

Comment:
another thing is installation too. we should think about how to pass the auth when specifying images with `install.image` :thinking:

Reply:
yeah, we currently dont have those either